app/pods/lack-of-public-apis/controller.js

```diff
@@ -1,5 +1,13 @@
 import Controller from '@ember/controller';
+import { readOnly } from '@ember/object/computed';
 
 export default Controller.extend({
+  _number: readOnly("number"),
```

Comment:
There's a problem with this line. 😬 

You have created a private property `_number`, but you're not using it anywhere, so that it does not make any difference.

The idea was to make the original property `number: Math.random()` private by renaming it into `_number`, and then creating a new property `number: readOnly("_number")` which is gonna become public API.

This way, consumers of the `number` property (e. g. other components) will not be able to overwrite it with a different value, thanks to `readOnly`.